src/documentation/fares_internal_format.md

```diff
@@ -0,0 +1,35 @@
+# Representing fares in Navitia Transit Model
+## Overview
+This document describes how fare structures are modeled in Navitia Transit Model. The internal model is composed of the followning objects:
+- Ticket
+- Rules
+
+## Model details
+### Ticket
+This object contains the necessary information to specify a ticket.
+
+Property | Type | Required | Description
+--- | --- | --- | ---
+id | UUID | Yes | Unique identifier
+input_data_format | Enum | Yes | Source of the fare data, fixed value `netex_fares_nl`.
+input_ticket_id | String | No | Id of the ticket found in the source data.
```

Comment:
Can't we end up with multiple "codes" to attach to tickets like it's the case for NTFS (lines).
In which case a simple field would not be enough.